android/app/src/main/java/org/fossasia/openevent/adapters/DayScheduleAdapter.java

```diff
@@ -99,6 +105,45 @@ public void onBindViewHolder(DayScheduleViewHolder holder, int position) {
             holder.slotTrack.getBackground().setColorFilter(storedColor, PorterDuff.Mode.SRC_ATOP);
             holder.slotTrack.setText(sessionTrack.getName());
 
+            if(currentSession.isBookmarked()) {
```

Comment:
@iamareebjamal I will create the Issue as soon as it gets merged as the Issue doesn't exist in the codebase currently.